tcrconvert/utils.py

```diff
@@ -16,5 +16,5 @@ def get_example_path(file_name):
     '.../tcrconvert/examples/tenx.csv'
```

Comment:
need to have ` # doctest: +ELLIPSIS` again on the code line for this ellipsis

Reply:
Added in ab75385